packages/interface/src/features/applications/components/ReviewApplicationDetails.tsx

```diff
@@ -70,6 +70,8 @@ export const ReviewApplicationDetails = (): JSX.Element => {
 
         <ValueField required body={address} title="Created By" />
 
+        <ValueField required body={application.shortBio} title="Project description" />
```

Comment:
Maybe rename the title as `Short Description`? Since it's same as the title of `application.bio`